### PR TITLE
denser table

### LINKED ISSUE #289: Fit more p-value data on screen for Wilcoxon and ANOVA
Condense p-value table rows. The default number of visible rows could also be changed to the full amount to avoid unused empty screen space.

We could try using the data table view, but that does not enable sorting rows by column value so I think we should stick to the vuetify table for now.

### PR DESCRIPTION
**Pull request by sgratzl**
closes #289

image

### CODE REVIEW

File: web/src/components/vis/MetaboliteTable.vue [hunk 1]
```diff
@@ -22,6 +22,9 @@ export default {
   data() {
     return {
       format: format('.4e'),
+      pagination: {
+        rowsPerPage: -1,
+      },
```
Comment (waxlamp) [inline web/src/components/vis/MetaboliteTable.vue]: Does this just mean "as many pages as possible"?

File: web/src/components/vis/MetaboliteTable.vue [hunk 2]
```diff
@@ -34,17 +37,29 @@ export default {
...
-    td {{ props.item.Metabolite }}
-    td.text-xs-right(v-for="c in headers.slice(1)",
+    td.cell {{ props.item.Metabolite }}
+    td.cell.text-xs-right(v-for="c in headers.slice(1)",
         :class="isInteresting(props.item[c.value]) ? 'highlight' : ''")
       | {{ format(props.item[c.value]) }}
 </template>
 
-<style lang="scss" scoped>
+<style scoped>
```
Comment (waxlamp) [inline web/src/components/vis/MetaboliteTable.vue]: Why get rid of SCSS here?

File: web/src/components/vis/MetaboliteTable.vue [hunk 3]
```diff
@@ -34,17 +37,29 @@ export default {
...
 .highlight {
   background-color: #ffadad;
 }
+
+.main {
+  height: 0;
+}
+
+.main >>> tr {
```
Comment (waxlamp) [inline web/src/components/vis/MetaboliteTable.vue]: What does this selector do?

Comment (sgratzl) [reply web/src/components/vis/MetaboliteTable.vue]: https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors

File: web/src/components/vis/MetaboliteTable.vue (hunk 1 above)
Comment (sgratzl) [reply web/src/components/vis/MetaboliteTable.vue]: https://v15.vuetifyjs.com/en/components/data-tables

```js
{
  descending: boolean",
  page: number",
  rowsPerPage: number // -1 for All",
  sortBy: string",
  totalItems: number"
}
```

Comment (waxlamp) [reply web/src/components/vis/MetaboliteTable.vue]: A comment in our code might be a good idea.

File: web/src/components/vis/MetaboliteTable.vue (hunk 2 above)
Comment (sgratzl) [reply web/src/components/vis/MetaboliteTable.vue]: SCSS doesn't support the deep selector

File: web/src/components/vis/MetaboliteTable.vue [hunk 4]
```diff
@@ -34,17 +37,29 @@ export default {
 </script>
 
 <template lang="pug">
-v-data-table.elevation-1(:headers="headers", :items="items", disable-initial-sort,
-    item-key="Metabolite")
+v-data-table.elevation-1.main(:headers="headers", :items="items", disable-initial-sort,
```
Comment (subdavis) [inline web/src/components/vis/MetaboliteTable.vue]: Not really worth changing here, but I tend toward more descriptive class names.  `metabolite-table` for instance.

File: web/src/components/vis/MetaboliteTable.vue [hunk 5]
```diff
@@ -34,17 +37,29 @@ export default {
...
 }
+
+.main {
+  height: 0;
+}
+
+.main >>> tr {
+  height: 25px;
+}
+.main >>> td.cell {
```
Comment (subdavis) [inline web/src/components/vis/MetaboliteTable.vue]: I'm a big fan of this.  I usually end up using vuetify's internal class names and it bites me when they change.  I like that this is an alternative option.

**Pull request merged**